custom_dc/setup_gcp.md

```diff
@@ -3,5 +3,73 @@ layout: default
 title: System Setup
 nav_order: 1
 parent: Custom Data Commons
-published: false
+published: true
 ---
+
+## Overview
+
+Custom Data Commons is deployed in Google Cloud Platform (GCP). The follow
+manual presents how to install a custom Data Commons instance in an existing GCP
+project (with id `PROJECT_ID`).
+
+### Domain
+
+The installation can use an existing domain, or create a default domain
```

Comment:
nit: Add a note saying that this guide uses the default domain? Maybe contact DC team for how to set up custom with custom domains? 

custom domains are supported after this PR.
https://github.com/datacommonsorg/website/pull/2218

Reply:
Added note. Can you file a bug to add instructions on how to use an existing domain.